src/components/SetAndReSetPassword.vue

```diff
@@ -1,56 +1,114 @@
 <script setup >
 import { ref } from 'vue'
 import { useRoute } from 'vue-router'
+import jwtDecode from 'jwt-decode'
 
 const route = useRoute()
 const props = defineProps({
   formData: Object
 })
 
+const email = ref()
+email.value = jwtDecode(route.query.token).user.email
+
+console.log(email.value)
+
 const data = ref({})
 
 </script>
 
 <template>
-  <v-container>
-    <v-row class="text-center" justify="center">
-        <v-col cols="6">
-        <form>
-          <v-col>
-            <h1>{{props.formData.text}}</h1>
-          </v-col>
-          <v-col>
-            <v-text-field v-if="props.formData.text === 'Set Password' "
-              name="name"
-              label="Name"
-              id="name"
-              type="text"
-              v-model="data.name"
-              required />
-
-              <v-text-field
+  <v-form class="d-flex flex-column justify-center align-center h-screen">
+    <v-card elevation="0" class="w-25">
+      <v-card-text align="center" >
+          <v-icon size="77" color="info" icon="mdi-weather-hurricane" />
+      </v-card-text>
+      <v-card-title class="justify-center py-0">
+        <h3>  Administration Panel </h3>
+      </v-card-title>
+    </v-card>
+  <v-card class="ma-2 pa-2  rounded-xl cardShadow">
+    <v-card-text align="center">
+      <h4 class="m-4 " >{{props.formData.text}}</h4>
+
+      <v-text-field v-if="props.formData.text === 'Set Password' "
+       hide-details
+      density="compact"
```

Comment:
Seems like there is an issue with the number of spaces in this line. Did not the linter fix it?

Reply:
It is still incorrectly formatted.